crates/sparrow-runtime/Cargo.toml

```diff
@@ -69,6 +69,8 @@ tokio-util.workspace = true
 tonic.workspace = true
 tracing.workspace = true
 uuid.workspace = true
+object_store = { version = "0.5.5", features = ["aws", "gcp"] }
```

Comment:
Should add these to the workspace, and use `object_store.workspace = true` here. This lets us have the versions defined in one place and used consistently in all the sub crates.

Reply:
Added in the latest revision.